awscli/customizations/configure/set.py

```diff
@@ -46,10 +46,14 @@ def __init__(self, session, config_writer=None):
             config_writer = ConfigFileWriter()
         self._config_writer = config_writer
 
+    def _get_config_file(self, path):
+        config_path = self._session.get_config_variable(path)
+        return os.path.expanduser(config_path)
+
     def _run_main(self, args, parsed_globals):
         varname = args.varname
         value = args.value
-        section = 'default'
+        profile = 'default'
         # Before handing things off to the config writer,
         # we need to find out three things:
         # 1. What section we're writing to (section).
```

Comment:
Just a reminder to change the comment to this:
```suggestion
         # 1. What section we're writing to (profile).
```